lib/Backends/Interpreter/Interpreter.cpp

```diff
@@ -145,6 +145,13 @@ bool Interpreter::isOpSupported(Kinded::Kind opKind, ElemKind elementTy) const {
   return true;
 }
 
+bool Interpreter::shouldLower(Node *N) const {
```

Comment:
can you please add test to verify that nodes are not indeed lowered.

Reply:
So far, we can use the Xray model (@artemrakhov posted it in Glow group) to test it: with the unrolling grouped convolutions, the code size is around 4398572, and without this lowering, the code size is 86288.  Since  this model has a lot of Convs with group>1, and there should be 9000 nodes after lowering Convs.  Do you think it is OK? Otherwise, am I allowed to dump some message to indicate the node is not lowered ? Btw, I also traced the execution to verify it.

Reply:
@beicy: This could be a very simple unit test in `graphOptzTest.cpp` (or `graphTest.cpp`). Create simple graph with GroupConv, run `lower` for it, and make sure that number of nodes is the same. If we want to complicate things, maybe can lower same graph on OpenCL backend (which does lower GroupConv), and verify that number of nodes is greater (because lowering took place).

Reply:
@rdzhabarov @artemrakhov Thanks! Will create a separate PR for the test